python/dicebeard/__init__.py

```diff
@@ -121,10 +128,14 @@ def __init__(self, *args, **kwargs):
         # Report back to the user about their answer
         if answer == total:
             await self.sender.sendMessage(
-                '✅ Correct: {:.3}s'.format(timer.total_time))
+                'Correct: {:.3}s'.format(timer.total_time))
+            return timer.total_time
+            
         else:
             await self.sender.sendMessage(
-                '❌ Wrong: {:.3}s'.format(timer.total_time))
+                'Wrong: {:.3}s'.format(timer.total_time))
+            return 10     
```

Comment:
If we `return None` we can do both; it defers the decision to `self.train_many` (which is the only function that currently cares about `self.train`'s return value).